CONTRIBUTING.md

```diff
@@ -6,6 +6,8 @@ generated using the pre-commit hooks for
 * [https://github.com/norwoodj/helm-docs](https://github.com/norwoodj/helm-docs)
 * [https://github.com/Lucas-C/pre-commit-hooks-nodejs](https://github.com/Lucas-C/pre-commit-hooks-nodejs)
 
+1. Install `helm-docs`: https://github.com/norwoodj/helm-docs#installation
```

Comment:
I _believe_ this is supposed to be handled by ASDF (also not documented) but I ran into issues with it myself... 

This seems like a good add for folks who don't want to install another dependency-management solution...

Reply:
oh yeah dunno about that, just following contribution instructions exactly 😜